### PR TITLE
feat(controller): add CRUD for finetune and space

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
add crud for SFT Space and SFT
sft validation dataset env set to run

### TODO

model copy with sft id
decouple datastore write(evaluation) from job create
add sft evaluation job
add model release status
add model generation relation

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/controller/src/main/java/ai/starwhale/mlops/api/protocol/sft/SftSpaceVo.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SftSpaceVo {
+    private Long id;
+    private String name;
+    private String description;
```
[the remark this answers is not in the document]
Comment (anda-ren) [reply server/controller/src/main/java/ai/starwhale/mlops/api/protocol/sft/SftSpaceVo.java]: done

File: server/controller/src/main/java/ai/starwhale/mlops/domain/ft/bo/FineTune.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class FineTune {
+    private Long id;
+    private Long spaceId;
+    private Long jobId;
+    private List<String> evalDatasets;
+    private List<String> trailDatasets;
```
Comment (goldenxinxing) [inline server/controller/src/main/java/ai/starwhale/mlops/domain/ft/bo/FineTune.java:28]: Does the 'trail' typo? train

Comment (anda-ren) [reply server/controller/src/main/java/ai/starwhale/mlops/domain/ft/bo/FineTune.java]: yes, it's a typo, fixed

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**